core/src/main/resources/hudson/logging/LogRecorder/configure.jelly

```diff
@@ -52,29 +52,21 @@ THE SOFTWARE.
                help="/help/LogRecorder/logger.html">
         <j:set var="instance" value="${null}" />
         <f:repeatable var="instance" items="${it.loggers}" name="loggers">
-          <table width="100%">
-            <tr>
-              <th style="white-space:nowrap;">
-                ${%Logger}
-              </th>
-              <td width="60%">
-                <f:textbox field="name" autoCompleteUrl="autoCompleteLoggerName" />
-              </td>
-              <th style="white-space:nowrap;">
-                ${%Log level}
-              </th>
-              <td>
-                <select name="level">
+          <div>
+            <f:entry title="${%Logger}">
+              <f:textbox  field="name" autoCompleteUrl="autoCompleteLoggerName"/>
+            </f:entry>
+            <f:entry title="${%Log level}">
+              <div class="jenkins-select">
+                <select class="jenkins-select__input" name="level">
                   <j:forEach var="lv" items="${it.LEVELS}">
-                    <f:option value="${lv.name}" selected="${instance.level==lv}">${lv.localizedName}</f:option>
+                    <f:option value="${lv.name}" selected="${instance.level==lv or (instance.level == null and lv == 'INFO')}">${lv.localizedName}</f:option>
```

Comment:
changed default level to INFO so that it doesn't default to an empty FINE log recorder, alternative or at least complementary to https://github.com/jenkinsci/jenkins/pull/4815

cc @Dohbedoh